.changeset/chilled-phones-explain.md

```diff
@@ -0,0 +1,6 @@
+---
+'@remote-dom/polyfill': patch
+'@remote-dom/core': patch
```

Comment:
I think I'd consider this a minor change for both. It does give you a capability you didn't have before, even if the default experience is implemented identically before and after the change.

Reply:
That makes sense. I'll make them a minor